index.html

```diff
@@ -24,22 +25,6 @@
                title="Bazel"
                alt="Bazel logo"
                class="logo img-responsive" />
-          <p class="tagline">{Fast, Correct} - Choose two</p>
```

Comment:
Thanks, Steren, and sorry for the misunderstanding. I think I should have explained the motivation for this change more clearly and emphasized that it was due to concerns around the visual design. I am glad we have reached an agreement on the layout of the hero and the placement of the summary and tagline. Thank you for being patient with me.